contracts/DomainController.sol

```diff
@@ -93,14 +96,16 @@ contract DomainController is Initializable, OwnableUpgradeable {
             revert InsufficientPayment();
 
```

Comment:
// 95 & 99 
// you read same thing twice, you can read it once and store it in a variable


// line 108 
// same thing as for line 95
// also you do not pay any rewards for contract owner 

// // - 2 points